tests/posix/xopen_streams/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(posix_xopen_streams)
```

Comment:
This should be `xsi_streams` to match the Open Group name which should presumably be `XSI_STREAMS` although the spec is not explicitly clear about that (the feature test macro is `_XOPEN_STREAMS`).

https://pubs.opengroup.org/onlinepubs/009695399/basedefs/xbd_chap02.html

So please adjust the testsuite to use `xsi_streams` instead of `posix_xopen_streams`, including the directory name too.

Thanks!